changelog.txt

```diff
@@ -17,6 +17,7 @@
 * Fix - Relax customer validation that was preventing payments from the pay for order page
 * Fix - Prevent the PMC migration to run when the plugin is not connected to Stripe
 * Fix - Fixes a fatal error in the OC inbox note when the new checkout is disabled
+* Fix - Allow checkout for logged-in users without an email in their account when a billing email is provided
```

Comment:
I merged from latest and this location is no longer correct -- the changelog entry needs to be move to 9.9.0.

Reply:
Thanks, moved in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4599/commits/8675b135e210918da8a06e50536a09512dc92549